src/main/frontend/search/protocol.cljs

```diff
@@ -1,8 +1,10 @@
 (ns ^:no-doc frontend.search.protocol)
 
 (defprotocol Engine
-  (query [this q option])
-  (rebuild-blocks-indice! [this])
+  (query [this q option]) 
+  (query-page [this q option])
```

Comment:
@xyhp915 Does this affect the plugins API?

Reply:
Plugin implementation from this protocol is OK. But the plugin SDK should be updated too. If this PR should be merged soon. I would create another PR to patch it!
<img width="1426" alt="CleanShot 2022-11-25 at 21 37 53@2x" src="https://user-images.githubusercontent.com/1779837/203997258-ca78a1a2-073c-427e-9895-fc5ea6bdb9aa.png">

Reply:
Cool, this PR has a small issue, I think we can include it for the coming 0.8.12 release.